src/rules/declaration-empty-line-before/index.js

```diff
@@ -36,6 +36,7 @@ export default function (expectation, options) {
         ignore: [
           "after-comment",
           "inside-single-line-block",
+          "after-declaration",
```

Comment:
Minor request, can we get these in alphabetical order please. It's helps when looking through lots of rule code.